travis.sh

```diff
@@ -4,11 +4,12 @@ set -euo pipefail
 
 function installTravisTools {
   mkdir -p ~/.local
-  curl -sSL https://github.com/SonarSource/travis-utils/tarball/v38 | tar zx --strip-components 1 -C ~/.local
+  curl -sSL https://github.com/SonarSource/travis-utils/tarball/v43 | tar zx --strip-components 1 -C ~/.local
   source ~/.local/bin/install
 }
 installTravisTools
 . installJDK8
+. installMaven35
```

Comment:
@alban-auzeill I changed `source ~/.local/bin/installMaven35` to simpler `. installMaven35`.